lib/sablon.rb

```diff
@@ -3,6 +3,7 @@
 
 require "sablon/version"
 require "sablon/configuration/configuration"
+require "sablon/relationship/relationship"
```

Comment:
I'm not sure if this file really needs nested in a directory but it doesn't hurt anything either.

Reply:
I agree with you, I will move the relationship.rb file to the sablon directory